geth-vm/configure.yml

```diff
@@ -0,0 +1,140 @@
+- hosts:
+    - {{ geth_instance }}-vm
+  gather_facts: no
+  vars_files:
+    - ../containers/versions.yml
+  vars:
+    geth_user:       "geth_user"
+    blockchain_path: "/home/{{ geth_user }}/blockchain"
+  tasks:
+    - become:      yes
+      become_user: root
+      block:
+        - fail: msg="Please supply correct geth_instance variable. They are only two option{{ ":" }} geth-testnet/geth-mainnet"
+          when: geth_instance != "geth-testnet" and geth_instance != "geth-mainnet"
+
+        - fail: msg="Please supply the \"ansible_host\" variable that should contain geth instance ip"
+          when:  ansible_host == ""
+
+        - name: Gathering Facts
+          setup:
+
+        - set_fact:
+            geth_flag:      "{% if geth_instance == 'geth-testnet' %} --rinkeby {% else %} {% endif %}"
+            ethereum_chain: "{% if geth_instance == 'geth-testnet' %}rinkeby{% else %}frontier{% endif %}"
```

Comment:
Please make it a dict instead of using `if`s. `geth-mainnet` and `geth-testnet` should be the keys.